src/SFA.DAS.EmployerIncentives.Web/Views/Payments/ListPayments.cshtml

```diff
@@ -165,11 +165,13 @@
                             </td>
                             <td data-label="First payment" class="govuk-table__cell">
                                 <partial name="_PaymentStatus" for="@apprenticeApplication.FirstPaymentStatus" />
+                                <partial name="_ClawbackStatus" for="@apprenticeApplication.FirstClawbackStatus" />
                             </td>
                             <td data-label="Second payment" class="govuk-table__cell">
                                 @if (apprenticeApplication.ShowSecondPaymentStatus)
                                 {
                                     <partial name="_PaymentStatus" for="@apprenticeApplication.SecondPaymentStatus" />
+                                    <partial name="_ClawbackStatus" for="@apprenticeApplication.SecondClawbackStatus" />
```

Comment:
so the ClawbackStatus with null prop values means the relevant Clawback doesn't actually exist? if so, it's not very obvious.